defaults/parameters.yaml

```diff
@@ -16,6 +16,7 @@ sanitize_metadata:
     - strain
     - name
   database_id_columns:
+    - "Accession ID"
```

Comment:
IIUC, this addition is necessary because the database id columns are now used for deduplication _before_ renaming (where `Accession ID=gisaid_epi_isl` below) but previously deduplicating was done _after_ renaming. It's not a behaviour change, as long as the original default renaming rule was left in place; otherwise it could be.

Would it be just as simple (or nearly so) to keep renaming as the first step (or very close to first) and thus leave the semantics of `database_id_columns` and `rename_fields` unchanged?

Either way, it'd be good to document in config and arg help text the order of operations.

Reply:
I found the script easier to understand with filtering of duplicates as early as possible. I'm not sure what my logic was in placing that step near the end originally...

But yes, good point about documenting these details. I've updated the script's help and the configuration guide in f4dc7c2 to clarify the order of operations.